packages/manager/modules/bm-server-components/src/general-information/inputs/inputs.class.js

```diff
@@ -0,0 +1,35 @@
+export default class Inputs {
```

Comment:
I have created a class because I wanted to have a shared function used within the server-installation-gabarit.controller as well as the server-installation-ovh.controller.